src/main/java/io/pravega/connectors/flink/table/catalog/pravega/PravegaCatalog.java

```diff
@@ -79,7 +80,7 @@
     private SerializationFormat serializationFormat;
 
     public PravegaCatalog(String catalogName, String defaultDatabase, String controllerUri, String schemaRegistryUri,
-                          String serializationFormat, String failOnMissingField, String ignoreParseErrors,
+                          String serializationFormat, Boolean failOnMissingField, Boolean ignoreParseErrors,
```

Comment:
If we need to change this, we would better change it separately in another PR, otherwise the behavior in 1.13 and 1.12 would be different which may bring confusion in the future.